ring/ring.go

```diff
@@ -61,17 +61,17 @@ func New(client CoordinationStateClient) *Ring {
 		quit:   make(chan struct{}),
 		done:   make(chan struct{}),
 		ingesterOwnershipDesc: prometheus.NewDesc(
-			"prometheus_distributor_ingester_ownership_percent",
+			"prism_distributor_ingester_ownership_percent",
 			"The percent ownership of the ring by ingester",
 			[]string{"ingester"}, nil,
 		),
 		ingesterTotalDesc: prometheus.NewDesc(
-			"prometheus_distributor_ingesters_total",
+			"prism_distributor_ingesters_total",
```

Comment:
Sorry, while you're at it: this shouldn't have a `_total` suffix, as it's a gauge, not a counter. Just `prism_distributor_ingesters`.